cmd/gsw/chainHelper.go

```diff
@@ -91,16 +91,16 @@ func workThreadAxtVcf(axtChannel <-chan *axt.Axt, ans chan<- *vcf.Vcf) {
 	close(ans)
 }
 
-func goFaChannel(ref []*fasta.Fasta) <-chan *fasta.Fasta {
+func goFaChannel(ref []*fasta.Fasta) <-chan fasta.Fasta {
```

Comment:
This is a performance change. One of Erics revelations was that when you pass a pointer through the channel the sending function i.e. ReadToChan sends it to the heap and not the stack therefore it cannot be efficiently removed by the garbage collector. I have seen this phenomenon in my own programs with vcf.GoReadToChan where the program will get slower as runtime increases. I think this is called a memory leak, but I don't know the CS lingo to be sure. 

I don't think this is very important fasta.GoReadToChan as fastas tend to be a small number of large structs (as opposed to VCF which is a large number of small structs) but for the sake of consistency I would like to make all GoReadToChan functions returns structs rather than pointers to structs (unless there is a good reason to keep it as pointers.

This particular change in GSW is a consequence of changing the fasta.GoReadToChan functions. I wanted to make sure I don't impact the functionality of GSW so as soon as a fasta record is read from the channel I convert it to a pointer for GSW to use. All calls of fasta.GoReadToChan in GSW are for input to the VariantGraph functions which is where I make the struct -> pointer conversion. 

I think Eric has benchmarks of this somewhere. I will see if I can find them, else make my own.